### PR TITLE
Make line text optional

### PR DESCRIPTION
**Pull request by Huquette**
https://github.com/GeotrekCE/Geotrek-admin/issues/3326

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -11,6 +11,7 @@ CHANGELOG
 - Enable filtering lists by objects IDs on APIv2 (#3458)
 - Add information desks link on Treks with AggregatorParsers
 - Add filter by manager to Blades module
+- Make line text optional (#3326)
```
[the remark this answers is not in the document]
Comment (Huquette) [reply docs/changelog.rst]: Done, thank you :)

**Review by marcantoinedupre — APPROVED** [review]

**Review by Chatewgne — APPROVED** [review]

File: geotrek/signage/models.py [hunk 2]
```diff
@@ -323,7 +323,7 @@ class Line(models.Model):
     number = models.IntegerField(verbose_name=_("Number"))
     direction = models.ForeignKey(Direction, verbose_name=_("Direction"), on_delete=models.PROTECT, null=True,
                                   blank=True)
-    text = models.CharField(verbose_name=_("Text"), max_length=1000)
+    text = models.CharField(verbose_name=_("Text"), max_length=1000, null=True, blank=True)
```
Comment (submarcos) [inline geotrek/signage/models.py:326]: Best practice is never allow Null value on charfield. use blank and default=""

File: geotrek/signage/templates/signage/blade_detail.html [hunk 3]
```diff
@@ -60,7 +60,7 @@ <h3>{% trans "Lines" %}</h3>
                 <tr>
                     <td>{{ line }}</td>
                     {% if is_direction_on_lines_enabled %}<td>{{ line.direction }}</td>{% endif %}
-                    <td>{{ line.text }}</td>
+                    <td>{{ line.text|default_if_none:""  }}</td>
```
Comment (Chatewgne) [inline geotrek/signage/templates/signage/blade_detail.html:63]: This is not necessary anymore now that field is `null=False, default=""`

**Review by Chatewgne — CHANGES_REQUESTED** [review]

File: geotrek/signage/templates/signage/blade_detail.html (hunk 3 above)
Comment (Huquette) [reply geotrek/signage/templates/signage/blade_detail.html]: Done, thank you :)

**Review by Chatewgne — APPROVED** [review]

**Pull request merged**